storybook/src/data/documents.data.js

```diff
@@ -1,7 +1,7 @@
 import uuid from 'uuid/v4';
 
 export default class DocumentBuilder {
-  static entityType = 'document';
+  entityType = 'document';
```

Comment:
Why is it non static? Didn't understand the reason

Reply:
static variables inside classes are still on proposal (stage 3): https://github.com/tc39/proposal-class-fields

Reply:
Since this syntax is still a proposal we have mainly two options:
- keep the static but change the syntax to a getter;
- drop, indeed, the static although it is in fact a static variable (or, at least, we would like it to be)

Reply:
A static getter would make sense and would be consistent with what we already do on other places (element declaration using class-based syntax, for example).

Reply:
Will do it.